tools/fuzzer/CMakeLists.txt

```diff
@@ -0,0 +1,54 @@
+project(SOF_FUZZER C)
+
+include(ExternalProject)
+
+set(source_directory "${PROJECT_SOURCE_DIR}/../testbench")
+
+ExternalProject_Add(sof_tb
+	SOURCE_DIR "${source_directory}"
+	PREFIX "${PROJECT_BINARY_DIR}/sof_tb"
+	BINARY_DIR "${PROJECT_BINARY_DIR}/sof_tb/build"
+	CMAKE_ARGS -DBUILD_LIBRARY=ON
```

Comment:
It looks like `BUILD_LIBRARY` is not used in testbench code, so no need to declaring it then.